app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -791,6 +819,29 @@ def test_update_experiment_publish_status(self):
         experiment = NimbusExperiment.objects.get(id=experiment.id)
         self.assertEqual(experiment.publish_status, NimbusExperiment.PublishStatus.REVIEW)
 
+    def test_update_live_rollout_publish_status(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create(
+            is_rollout=True,
+            publish_status=NimbusExperiment.PublishStatus.IDLE,
+            status=NimbusExperiment.Status.LIVE,
+        )
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "publishStatus": NimbusExperiment.PublishStatus.DIRTY.name,
+                    "changelogMessage": "test changelog message",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
+
+        experiment = NimbusExperiment.objects.get(id=experiment.id)
+        self.assertEqual(experiment.publish_status, NimbusExperiment.PublishStatus.DIRTY)
```

Comment:
Oh yeah, we could have the frontend be responsible for explicitly bumping it from IDLE to DIRTY, but I don't think there's any reason why that would be preferable to having the backend do it.  I'd imagine the backend would be responsible for making sure that an edit can only come through on LIVE(IDLE/DIRTY, is_rollout), and then the backend would bump from IDLE to DIRTY?  Then the frontend doens't need to worry about it?

Reply:
Totally agreed 🙌

Reply:
If the backend is the one setting this then we don't need to cover this case at all.  In fact I might go as far as to say the frontend CAN'T do this so you don't want up with accidentally setting the publish_status in the frontend in a way that conflicts with the backend doing it.